docs/global-settings.md

```diff
@@ -50,3 +50,29 @@ For using the [Content Delivery API](https://www.contentful.com/developers/docs/
 
 *Recommended way here is to set the `host` in the `metalsmith.json` or global config and overwrite it if needed in depending source files.*
 
+### 'filterTransforms' *(optional)*
+
+If you want to use dynamic values in a source file's filter query, you can provide an object containing named functions that will be invoked during the metalsmith build. For example, if you provide a configuration like this:
+
+```javascript
+{
+  filterTransforms: {
+    __NOW__() {
+      return (new Date()).toISOString();
+    },
+  }
+```
```

Comment:
here is one curly missing. ;)

But as you're using this, I assume you're not using the `metalsmith.json` approach, right?

A lot of people might use a simple `JSON` file as configuration and inside of that, this wouldn't be valid. Can we elaborate a little bit there? and make this clearer?